### PR TITLE
feat(wallet): add `user_id` column to `crypto_wallet_signature`

### PR DESCRIPTION
**Pull request by robertu7**

### CODE REVIEW

**Review by jeremyOk — APPROVED** [review]

File: src/mutations/user/putWallet.ts [hunk 1]
```diff
@@ -96,7 +96,7 @@ const resolver: MutationToPutWalletResolver = async (
     wallet = await atomService.update({
       table,
       where: { id, userId: viewer.id },
-      data: { address },
+      data: { address, updatedAt: new Date() },
```
Comment (devformatters) [inline src/mutations/user/putWallet.ts:99]: `where : { ..., archived: false }` ?

**Review by devformatters — APPROVED** [review]
Nice. work. 🐷

File: src/mutations/user/putWallet.ts (hunk 1 above)
Comment (robertu7) [reply src/mutations/user/putWallet.ts:99]: already did the [checking before](https://github.com/thematters/matters-server/blob/151ce56079d809497ead73fe2fc6f7ed04b62af1/src/mutations/user/putWallet.ts#L87-L94), should be good?

Comment (devformatters) [reply src/mutations/user/putWallet.ts:99]: but this is going to update both `archived = true or archived = false`

**Pull request merged**